resources/tenant-onboarding-app.yaml

```diff
@@ -229,6 +229,10 @@ Parameters:
   OnboardingDdbTable:
     Description: Internal DDB table used to store Onboarding Metadata
     Type: String
+  S3ExtensionBucket:
```

Comment:
Other naming options that kinda match existing patterns better
`ObjectStorageBucket` `StorageBucket` `ObjectsBucket`